src/VisualStudio/IntegrationTest/TestUtilities/InProcess/SolutionExplorer_InProc.cs

```diff
@@ -888,25 +888,36 @@ public void OpenFile(string projectName, string relativeFilePath)
             ErrorHandler.ThrowOnFailure(textManager.NavigateToLineAndColumn(textLines, VSConstants.LOGVIEWID.Code_guid, line, column, line, column));
         }
 
-        public void CloseFile(string projectName, string relativeFilePath, bool saveFile)
+        public void CloseDesignerFile(string projectName, string relativeFilePath, bool saveFile)
         {
-            var document = GetOpenDocument(projectName, relativeFilePath);
-            if (saveFile)
-            {
-                SaveFileWithExtraValidation(document);
-                document.Close(EnvDTE.vsSaveChanges.vsSaveChangesYes);
-            }
-            else
+            CloseFile(projectName, relativeFilePath, VSConstants.LOGVIEWID.Designer_guid, saveFile);
+        }
+
+        public void CloseCodeFile(string projectName, string relativeFilePath, bool saveFile)
+        {
+            CloseFile(projectName, relativeFilePath, VSConstants.LOGVIEWID.Code_guid, saveFile);
+        }
```

Comment:
nit: you can inline this and above

Reply:
I'm going to keep it like this since it's causing build failures and we already have a passing build in the current form.